### DISCUSSION TITLE
How to sequentially fold instead of simultaneously folding all folds?

### DISCUSSION THREAD

**Discussion by dynomighty** [amandaghassaei/OrigamiSimulator#60] [Q&A] [answered]
![microgami photos outside 2](https://github.com/amandaghassaei/OrigamiSimulator/assets/5889393/62505641-486c-4083-9bdd-d4e21ede7dc1)

Hello everyone, 

I'm an artist and not a programer but I've managed to create something close to my design but I'm running into problems now and could really use some help.

Basically I'm hoping to use the Origami Simulator to submit my origami wallet concept to the [Red Dot Design award competition](https://www.red-dot.org/design-concept/participate) in 7 days. I wanted to use the Origami Simulator to demonstrate the folding method for the wallet but I'm having a problem. 

In the traditional design pattern there are a series of folds that are flipped after being creased but this type of sequential folding is apparently not possible with the Origami Simulator.  

Essentially the same linear line that defines a mountain or valley fold is at one point on the line a valley fold and at the other point a mountain fold. This contradiction is something that the simultaneous folding sequence can't adjust for so my model is jittering while trying to resolve these conflicts and results in a jumbled mess of pointing angles. 

This is causing issues with the folds conflicting at certain points in the process. 
Here is a video of the effect: [https://youtu.be/kDXMTlPWJrY](https://youtu.be/kDXMTlPWJrY)

I know that other origami patterns have this same issue but you must have found a solution - for example the Swan - am I supposed to use the yellow lines in my design? If so, where should those go? 

Is there anyone I can reach out to for help with this? 

Thanks for any help

**Comment by dynomighty** [comment]
![MicroGami folding 2023 Wallet - Test Print](https://github.com/amandaghassaei/OrigamiSimulator/assets/5889393/f6bd04fb-a90b-477e-97e9-474efe07a203)

**Comment by amandaghassaei** [comment, marked as answer]
I've just added this capability in my new project at packcad.com (a successor to origami simulator):

https://github.com/user-attachments/assets/e6f006e4-43bf-4783-aeba-2fecb574b571

You can find this setup in Examples>Folding Simulation>Mailer Box Sequence in the PackCAD app.
